Win11ThemeTest/TextBoxTest.cs

```diff
@@ -0,0 +1,449 @@
+            Keyboard.Release(FlaUI.Core.WindowsAPI.VirtualKeyShort.CONTROL);
+            Wait.UntilInputIsProcessed(TimeSpan.FromMilliseconds(1000));
+
+            textBox.RightClick();
+            Wait.UntilInputIsProcessed(TimeSpan.FromMilliseconds(2000));
+
+            var cutText = TextWindow.FindFirstDescendant(cf => cf.ByName("Cut")).AsMenuItem();
+            Wait.UntilInputIsProcessed(TimeSpan.FromMilliseconds(1000));
+            Assert.IsNotNull(cutText);
+            
+            Assert.IsTrue(cutText.IsEnabled);
+            Wait.UntilInputIsProcessed(TimeSpan.FromMilliseconds(1000));
+
+            cutText.Click();
+            Wait.UntilInputIsProcessed();      
+
+        }
+
+        [Test]
+        public void tbb11_rightClickTest_Copy()
+        {
+            textBox = TextWindow.FindFirstDescendant(cf => cf.ByAutomationId("tbTxt")).AsTextBox();
+
+            textBox.Enter("Hello World!Hello World!Hello World!Hello World!Hello World!Hell World!Hello World!Hello World!Hello World!");
+            Wait.UntilInputIsProcessed(TimeSpan.FromMilliseconds(1000));
+
+            Keyboard.Press(FlaUI.Core.WindowsAPI.VirtualKeyShort.CONTROL);
+            Keyboard.Press(FlaUI.Core.WindowsAPI.VirtualKeyShort.KEY_A);
+           
+            Wait.UntilInputIsProcessed(TimeSpan.FromMilliseconds(2000));
+            Keyboard.Release(FlaUI.Core.WindowsAPI.VirtualKeyShort.KEY_A);
+            Keyboard.Release(FlaUI.Core.WindowsAPI.VirtualKeyShort.CONTROL);
+
+            //Assert.IsNotNull(textBox);
+            textBox.RightClick();
+            Wait.UntilInputIsProcessed(TimeSpan.FromMilliseconds(2000));
+
+            var pasteText = TextWindow.FindFirstDescendant(cf => cf.ByName("Copy")).AsMenuItem();
+            Wait.UntilInputIsProcessed(TimeSpan.FromMilliseconds(3000));
+            pasteText.GetClickablePoint();
+            Assert.IsNotNull(pasteText);
+            pasteText.Click();
+            Wait.UntilInputIsProcessed();
+           
+        }
+
+        [Test]
+        public void tbb10_rightClickTest_Paste()
+        {
+            textBox = TextWindow.FindFirstDescendant(cf => cf.ByAutomationId("tbTxt")).AsTextBox();
+
+            textBox.Enter("Hello World!Hello World!Hello World!Hello World!Hello World!Hell World!Hello World!Hello World!Hello World!");
+            Wait.UntilInputIsProcessed(TimeSpan.FromMilliseconds(1000));
+
+            textBox.RightClick();
+            Wait.UntilInputIsProcessed(TimeSpan.FromMilliseconds(2000));
+
+            var pasteText = TextWindow.FindFirstDescendant(cf => cf.ByName("Paste")).AsMenuItem();
+            Wait.UntilInputIsProcessed(TimeSpan.FromMilliseconds(1000));
+            Assert.IsNotNull(pasteText);
+            pasteText.Click();
+        }
+
+        //Confirm that the entered text is retained when navigating away and returning to the page.
+
+
+        #endregion
+
+        #region PostiveTests
+        /* Positive  Test Scenarios
```

Comment:
Implementation is pending for this

Reply:
Removed as this scenario is not applicable.